src/app/booking/useDotButton.js

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import {IconButton} from "@mui/material";
+import {Box} from "@mui/system";
+import {useTheme} from "@mui/material/styles";
+
+export const useDotButton = (emblaApi) => {
+  const [selectedIndex, setSelectedIndex] = useState(0)
+  const [scrollSnaps, setScrollSnaps] = useState([])
+
+  const onDotButtonClick = useCallback(
+    (index) => {
+      if (!emblaApi) return
+      emblaApi.scrollTo(index)
+    },
+    [emblaApi]
+  )
+
+  const onInit = useCallback((emblaApi) => {
+    setScrollSnaps(emblaApi.scrollSnapList())
+  }, [])
+
+  const onSelect = useCallback((emblaApi) => {
+    setSelectedIndex(emblaApi.selectedScrollSnap())
+  }, [])
+
+  useEffect(() => {
+    if (!emblaApi) return
+
+    onInit(emblaApi)
+    onSelect(emblaApi)
+    emblaApi.on('reInit', onInit)
+    emblaApi.on('reInit', onSelect)
+    emblaApi.on('select', onSelect)
+  }, [emblaApi, onInit, onSelect])
+
+  return {
+    selectedIndex,
+    scrollSnaps,
+    onDotButtonClick
+  }
+}
+
+export const DotButton = ({ isSelected, onClick, dotIconButtonSx, dotBoxSx }) => {
+  const theme = useTheme();
+
+  return (
+    <IconButton onClick={onClick} size="small" sx={dotIconButtonSx}>
```

Comment:
For a god sake. It should be rounded
<img width="218" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/3b48fd39-d27c-4110-9aae-787fb2e796b1">